manifests/c/Canon/GPCL6_V4_PrinterDriver_V21/00/2.72/Canon.GPCL6_V4_PrinterDriver_V21.00.locale.en-EU.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Canon.GPCL6_V4_PrinterDriver_V21.00
+PackageVersion: 2.72
+PackageLocale: en-EU
+Publisher: Canon
+PublisherUrl: https://my.canon/en/landing
+PublisherSupportUrl: https://my.canon/en/support
+PrivacyUrl:   https://my.canon/privacy
+PackageName: Canon PCL6 Driver
+PackageUrl: https://my.canon/en/support/0100952701?model=3617B006
+License: MIT License
+ShortDescription: Installation file for Canon PCL6 Drivers version 2.72, dated March 16th, 2023

```

```suggestion
ShortDescription: Installation file for Canon PCL6 Drivers
```
The version and date should be omitted from a static field